src/kontrol/foundry.py

```diff
@@ -400,11 +400,11 @@ def loc_FOUNDRY_FAILED() -> KApply:  # noqa: N802
 
     @staticmethod
     def symbolic_contract_prefix() -> str:
-        return 'CONTRACT'
+        return 'C'
 
     @staticmethod
-    def symbolic_contract_id() -> str:
-        return Foundry.symbolic_contract_prefix() + '_ID'
+    def symbolic_contract_id(contract_name: str) -> str:
+        return Foundry.symbolic_contract_prefix() + '_' + contract_name.upper() + '_ID'
```

Comment:
This allows us to have more informative variable names and a better debugging experience, while also preventing variable capture.